pom.xml

```diff
@@ -34,10 +34,28 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <jakarta.json.version>2.0.0-RC1</jakarta.json.version>
+        <jakarta.json.version>2.0.0-RC2</jakarta.json.version>
         <jakarta.json.bind.version>2.0.0-RC1</jakarta.json.bind.version>
+        <jakarta.enterprise.cdi-api.version>3.0.0-M2</jakarta.enterprise.cdi-api.version>
         <netbeans.hint.jdkPlatform>JDK_9</netbeans.hint.jdkPlatform>
     </properties>
+    
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>jakarta.json</groupId>
+                <artifactId>jakarta.json-api</artifactId>
+                <version>${jakarta.json.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.glassfish</groupId>
+                <artifactId>jakarta.json</artifactId>
+                <version>${jakarta.json.version}</version>
+                <classifier>module</classifier>
+                <scope>compile</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```

Comment:
Can you remove the dependencyManagement and just have the regular `<dependencies>`. No need for dependency management on a single-module project